package.json

```diff
@@ -10,6 +10,7 @@
     "deploy": "docusaurus deploy",
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
+    "lint:md": "markdownlint --ignore node_modules/ \"./docs/**/*.md\"",
```

Comment:
Should we add a fix option as well?

"lint:md:fix": "markdownlint --fix --ignore node_modules/ \"./docs/**/*.md\"",